marc_to_solr/translation_maps/location_display.rb.tmpl

```diff
@@ -21,6 +20,8 @@
  "eastasian$ref"=>"East Asian Library",
  "eastasian$reserve"=>"East Asian Library",
  "eastasian$UNASSIGNED"=>"East Asian Library",
+ "online$etasrcp"=>"Electronic Access",
```

Comment:
ETAS came back?

Reply:
@mzelesky do we need this here for some legacy purpose? I can't think on any.

Reply:
I'll remove it. It appears in this file because it is still in Alma.